lib/newsletter_exclusion.rb

```diff
@@ -0,0 +1,93 @@
+module NewsletterExclusions
```

Comment:
The lib folder is in load_path and but the file is named newsletter_exclusion and the module NewsletterExclusions (with the 's'). Are you sure that is working? But working or not I would rename the file to newsletter_exclusions.rb (with 's'). :)